gst/nnstreamer/include/tensor_typedef.h

```diff
@@ -75,6 +80,20 @@
      * "dimension = (string) dim1:dim2:dim3:dim4"
      */
 
+/**
+ * @brief Caps string for the caps template of tensor stream.
+ * format should be a string that describes the data format, or possible formats of incoming tensor.
+ *
+ * If the data format is static, another tensor information should be described in caps.
+ * - num_tensors: The number of tensors in a GstBuffer.
+ * - types: The list of data type in the tensor. type should be one of types in GST_TENSOR_TYPE_ALL. (types=(string)"type1,type2,type3")
+ * - dimensions: The list of dimension in the tensor. dimension should be a formatted string with rank NNS_TENSOR_RANK_LIMIT. (dimensions=(string)"dim1:dim2:dim3:dim4,dim1:dim2:dim3:dim4")
```

Comment:
`,` should be avoided for caps description. I think it can't be parsed.
How about `(dimensions=(string)"dim1:dim2:dim3:dim4.dim1:dim2:dim3:dim4")`

Reply:
now we use comma for this (e.g., gst_tensors_info_get_types_string() https://github.com/nnstreamer/nnstreamer/blob/f5dd59fc9af9d3c37d1c597b7af97bf58ad61fda/gst/nnstreamer/tensor_common.c#L669)

Reply:
Ah.. using with the quotation mark. Thanks.